CONTRIBUTING.md

```diff
@@ -33,16 +33,16 @@ using the `jetbrains-ide` & `team/integrations` labels.
 | Reformat Java and Kotlin sources                              | `./gradlew spotlessApply`                                                |
 | Debug agent JSON-RPC communication                            | `tail -f build/sourcegraph/cody-agent-trace.json`                        |
 
-## Using Alpha channel releases
+## Using Nightly channel releases
 
-We occasionally publish plugins to the "Alpha" channel instead of the default
-"Stable" channel. The alpha channel is primarily intended to publish
+We occasionally publish plugins to the "Nightly" channel instead of the default
```

Comment:
And it will be possible to change it from the `Sourcegraph & Cody` settings too 😄